app/javascript/react/components/PizzaShowContainer.js

```diff
@@ -24,6 +28,7 @@ const PizzaShowContainer = (props) => {
       const new_pizza = parsedPizza.pizza
       new_pizza.brand = parsedPizza.brand.name
       new_pizza.brand_id = parsedPizza.brand.id
+      new_pizza.reviews = parsedPizza.reviews
```

Comment:
We may have allowed this earlier, but you shouldnt need to deconstruct all of these attributes. Please use a serializer to return this information to the frontend in an already organized data structure such that you could just call `setPizza(parsedPizza)`